admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -1213,4 +1226,23 @@ class AdminModuleController {
   }
 }
 
+const BOEvent = {
```

Comment:
I don't think it's a good place for such code. Can't we use the event emitter that we already have in BO?

Reply:
I can use the eventEmitter component but it means do the same for all the events https://github.com/PrestaShop/PrestaShop/pull/31285/files#diff-e7328df916ab4f26f79b085040bfe2a0e1c5a4b9f1564365371b78df89ff17d2R228

That means more code, more QA :D :D

Reply:
ok, I didn't notice that you had to copy the code from one file to another

do you think you could export this code and import in both places? 🤔